src/main/java/org/prebid/server/bidder/openx/OpenxBidder.java

```diff
@@ -226,10 +226,16 @@ private ExtPrebid<ExtImpPrebid, ExtImpOpenx> parseOpenxExt(Imp imp) {
         return impExt;
     }
 
-    private ObjectNode makeImpExt(Map<String, JsonNode> customParams, ExtImpAuctionEnvironment auctionEnvironment) {
-        return customParams != null || auctionEnvironment != ExtImpAuctionEnvironment.SERVER_SIDE_AUCTION
-                ? mapper.mapper().valueToTree(OpenxImpExt.of(customParams, auctionEnvironment))
-                : null;
+    private ObjectNode makeImpExt(ExtPrebid<ExtImpPrebid, ExtImpOpenx> impExtPrebid, Imp imp) {
```

Comment:
What if

```java
 private ObjectNode makeImpExt(ExtPrebid<ExtImpPrebid, ExtImpOpenx> impExtPrebid, ObjectNode impExt) {
        final ObjectNode updatedExt = impExt.deepCopy();
        updatedExt.remove(IMP_EXT_SKIP_FIELDS);
        final Map<String, JsonNode> customParams = impExtPrebid.getBidder().getCustomParams();
        if (MapUtils.isNotEmpty(customParams)) {
            updatedExt.set(CUSTOM_PARAMS_FIELD_NAME, mapper.mapper().valueToTree(customParams));
        }

        return updatedExt;
    }
```

Reply:
Then no need of OpenxImpExt class

Reply:
I agree, if we don't need to keep the OpenxImpExt for exposing the params, it can be simpler.
I pushed a change along these lines that also skips the encoding by copying customParams from imp.ext.bidder, let me know how it looks.